### PR TITLE
Provide TypeScript definitions file

### PR DESCRIPTION
**Pull request by joeytwiddle**
Useful for anyone using this package in a TypeScript project.

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+interface CIDRTools {
+  merge(networks: Networks): Network[];
+
+  exclude(baseNetworks: Networks, excludeNetworks: Networks): Network[];
+
+  expand(networks: Networks): Network[];
+
+  overlap(networksA: Networks, networksB: Networks): boolean;
+
+  normalize(Network): Network;
```
Comment (silverwind) [inline index.d.ts]: I think you can do without the empty lines between the methods.

**Comment by silverwind** [comment]
Do these need to be published? If so, please add the file to the `files` array.

File: index.d.ts (hunk 1 above)
Comment (joeytwiddle) [reply index.d.ts]: Done (rebased).

Great tool by the way. I needed to turn an acceptlist into a rejectlist:

```js
const rejectList = cidrTools.exclude('0.0.0.0/0', acceptList);
```

(Works the same the other way too.)

Comment (silverwind) [reply index.d.ts]: Care to answer my other question in the issue comments? I have no experience with TypeScript, but I guess definitions must be published along with the module so tools can pick them up from `node_modules`?

**Comment by joeytwiddle** [comment]
Yes, good catch! I have added `index.d.ts` to `files`.

**Pull request merged**